### DISCUSSION TITLE
Review requests notification in Slack integration

### DISCUSSION THREAD

**Discussion by biccio** [GitbookIO/community#133] [Feature Requests]
### What's your feature/idea?

It would be useful to have the Gitbook review requests included in Slack integration notifications.

### What's your desired outcome?

I'd like to receive notifications of Gitbook review requests in Slack.

### What's the impact of this for you?

- [ ] Critical
- [x] High
- [X] Medium
- [ ] Low

**Comment by MantisClone** [comment]
:+1: Agreed! I want to be notified in Slack when a Gitbook review is requested of me.

**Comment by jonat123** [comment]
I agree with the above 👍

It would be super useful if it was possible to get notified whenever someone adds a change request for review, as well if I get added as a reviewer to a change request! 😄

**Comment by JaniceManwiller** [comment]
Especially since recently the GitBook review requests and comments on my requests seem to keep landing in the Spam box, even when I add GitBook as a contact.
